### PR TITLE
New space form and tests

### PR DESCRIPTION
**Pull request by fbell123**
Also updated README

### CODE REVIEW

**Comment by fbell123** [comment]
@lauraweston @MrChristoff

File: app/views/new_space.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>New listing</title>
+  </head>
```
Comment (lauraweston) [inline app/views/new_space.erb:6]: We shouldn't need this now we have the layout.erb

File: spec/features/new_space_spec.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+feature 'Add a new space' do
+  scenario 'A host can view the page to add a new space' do
+    visit '/spaces/new'
+    expect(page.status_code).to eq 200
```
Comment (lauraweston) [inline spec/features/new_space_spec.rb]: This expect is covered in the route tests, can we remove it here?

File: spec/web_helpers.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+def new_user
+  # INSERT INTO users (created_at, name, email, phone_number, bio)
+  # VALUES ('2016-10-31 17:57:59 +0000','John','john@email.com','07293728298','Lovely man');
```
Comment (lauraweston) [inline spec/web_helpers.rb]: Delete comments

**Review by lauraweston — CHANGES_REQUESTED** [review]
See comments

**Pull request merged**